docs/getting-started/installation/mac.md

```diff
@@ -131,16 +87,48 @@ To force the platform to use `ruby`, you can either:
 	 test and development, use a file named .env.local, or modify .env.test.local
 	 and .env.development.local.
 
-5. Run `bin/setup`
+4. Do final application setup (including the installation of requisite Ruby gems and NodeJS modules) with `./bin/setup`.
+
+## Start Up Your Forem!
+
+```sh
+./bin/startup
+```
+
+Presuming no errors here, your Forem should be accessible at `http://localhost:3000`.
+
+To run unit tests, first, prepare the database:
+
+```sh
```

Comment:
Based on https://forem-team.slack.com/archives/C0134R988P9/p1695157381316579?thread_ts=1695154032.277929&cid=C0134R988P9 sounds like we just need to only run `bin/rails db:setup`?

Reply:
When I run `./bin/rails db:setup` I only see the development DB get populated, not a test DB :/

Reply:
That's odd. It doesn't say it is but  `bin/setup`, which also runs `bin/rails db:setup` in it, will cover the test db setup.

Reply:
For the sake of merging this sooner than later, I'm going to stick with what's written here, and we can trim down in a following PR as desired.